src/main/java/com/icthh/xm/tmf/ms/communication/domain/spec/EmailTemplateSpec.java

```diff
@@ -20,6 +21,7 @@ public class EmailTemplateSpec {
     private String contextExample;
     private String contextSpec;
     private String contextForm;
+    private List<String> dependsOnTemplateKeys;
```

Comment:
add getter that set dependsOnTemplateKeys in case of null

Reply:
added

Reply:
add smth other, but not method that SET dependsOnTemplateKeys in case of null